python/ray/tests/test_state_api.py

```diff
@@ -3080,18 +3078,14 @@ def ready(self):
 
     # Make sure when the --detail option is specified, the default formatting
     # is yaml. If the format is not yaml, the below line will raise an yaml exception.
-    print(
-        yaml.load(
-            result.output,
-            Loader=yaml.FullLoader,
-        )
-    )
+    # Retrieve yaml content from result output
+    print(yaml.safe_load(result.output.split("---")[1].split("...")[0]))
 
     # When the format is given, it should respect that formatting.
-    result = runner.invoke(ray_list, ["actors", "--detail", "--format=table"])
+    result = runner.invoke(ray_list, ["actors", "--detail", "--format=json"])
```

Comment:
This is an intended test I believe. We should not be changing this.

Reply:
I see the test motivation is to respect `--format`.

With JSON, the test would catch the issue if the format option isn't honored (default is yaml). 
With current code (or moving to safe_yaml), the test always succeeds as it raises pyYaml exception.